packages/vscode-extension/src/webview/views/PreviewView.tsx

```diff
@@ -341,7 +341,12 @@ function PreviewView() {
           tooltip={{
             label: "Select an element to inspect it",
           }}
-          onClick={() => setIsInspecting(!isInspecting)}
+          onClick={() => {
+            sendTelemetry("inspector:inspector-button-clicked", {
```

Comment:
i'm thinking whether `inspector:enabled` would be better but there are different ways to enable inspector so idk 🤔 

just 

`inspector:button-clicked` would be enough tho